ferrocene/doc/user-manual/src/targets/x86_64-pc-windows-msvc.rst

```diff
@@ -0,0 +1,34 @@
+.. SPDX-License-Identifier: MIT OR Apache-2.0
+   SPDX-FileCopyrightText: The Ferrocene Developers
+
+.. _x86_64-pc-windows-msvc:
+
+:target:`x86_64-pc-windows-msvc`
+================================
+
+The ``x86_64-pc-windows-msvc`` Ferrocene target provides support for Host as
+well as Cross-compilation for Microsoft Windows on x86_64 using the native
+ABI (MSVC).
+
+Prerequisites
+-------------
+
+You need to install Microsoft Build Tools either via
+`installing Visual Studio <https://visualstudio.microsoft.com/downloads/>`_
+or installing the Build Tools directly via the same installer.
```

Comment:
Not quite that simple. We need to capture some additional nuances. 

While rustup just suggests using VSCode build tools, many professional customers (such as those who read our docs) are not able to do that unless they have a Visual Studio license.


In many cases, our users are not actually developing for Windows itself... but some arm eabi or (one day) qnx etc.


In those cases, we believe it may be sufficient to use our llvm-tool 's included llvm-link.exe , rust-lld , or some other tool.


In addition, some specific build contexts may be able to use the Windows SDK to provide the necessary libraries.